package.json

```diff
@@ -20,19 +20,43 @@
     "uploadCoverage": "gulp uploadCoverage"
   },
   "lint-staged": {
-    "*.{js,ts,json}": ["prettier --single-quote --print-width 140 --write", "git add"]
```

Comment:
Is it possible that you ran prettier without the printWidth: 140 parameter? Maybe with an extension in your editor? This seems to be less than 140 characters. It is possible to add this config in the package.json like this
```  "prettier": {
    "printWidth": 140,
    "singleQuote": true
  }```

I will add it in my next PR but if you need it now, might as well.

Reply:
I think it might be vscode auto formatting ? 

Not sure why it's not getting reformatted properly with the precommit hook though 🤔